### PR TITLE
Update Parsoid content-type to 1.2.1

### PR DESCRIPTION
**Pull request by gwicke**
Also:

- Include charset parameter, as in the response.
- Link to the mediawiki.org documentation page.

### CODE REVIEW

**Comment by Pchelolo** [comment]
But was the parsoid response already updated? If the the tests should fail.

**Comment by gwicke** [comment]
The corresponding Parsoid change is https://gerrit.wikimedia.org/r/#/c/279458/. We should sync this with their change.

File: config.test.yaml [hunk 1]
```diff
@@ -67,8 +67,8 @@ wiktionary_project: &wiktionary_project
 # Hacky way to parametrize RESTBase tests. TODO: Move to config?
 test:
   content_types:
-    html: text/html; charset=utf-8; profile="mediawiki.org/specs/html/1.2.0"
-    data-parsoid: application/json; charset=utf-8; profile="mediawiki.org/specs/data-parsoid/0.0.2"
+    html: text/html; charset=utf-8; profile="https://www.mediawiki.org/wiki/Specs/HTML/1.2.1"
+    data-parsoid: application/json; charset=utf-8; profile="https://www.mediawiki.org/wiki/Specs/data-parsoid/0.0.2"
     wikitext: text/plain; charset=utf-8; profile="mediawiki.org/specs/wikitext/1.0.0"
```
Comment (d00rman) [inline config.test.yaml]: Needs to be changed as well in the same manner as the two profiles above.

File: config.test.yaml [hunk 2]
```diff
@@ -67,9 +67,9 @@ wiktionary_project: &wiktionary_project
 # Hacky way to parametrize RESTBase tests. TODO: Move to config?
 test:
   content_types:
-    html: text/html; charset=utf-8; profile="mediawiki.org/specs/html/1.2.0"
-    data-parsoid: application/json; charset=utf-8; profile="mediawiki.org/specs/data-parsoid/0.0.2"
-    wikitext: text/plain; charset=utf-8; profile="mediawiki.org/specs/wikitext/1.0.0"
+    html: text/html; charset=utf-8; profile="https://www.mediawiki.org/wiki/Specs/HTML/1.2.1"
+    data-parsoid: application/json; charset=utf-8; profile="https://www.mediawiki.org/wiki/Specs/data-parsoid/0.0.2"
+    wikitext: text/plain; charset=utf-8; profile="https://www.mediawiki.org/Specs/wikitext/1.0.0"
```
Comment (d00rman) [inline config.test.yaml]: Heh, it's `wiki/Specs` :P

**Comment by d00rman** [comment]
Damn, now failing with unrelated tests... Rebase might help?

**Comment by gwicke** [comment]
The failures look like mobile app deployment-prep failures to me. We debugged this yesterday, their config.yaml needs to be changed like this: https://gist.github.com/gwicke/eb318c3d12f8988b0949

**Comment by gwicke** [comment]
The next Parsoid deploy is planned for tomorrow, at which point this PR should make our tests pass again.

**Comment by gwicke** [comment]
Actually, judging by the latest test results it looks like the Parsoid content-type changes are already live.

**Comment by gwicke** [comment]
In any case, those failures are not related to this patch, so I'd say lets get this one merged.

**Pull request merged**